packages/url_launcher/url_launcher_web/README.md

```diff
@@ -13,3 +13,13 @@ should add it to your `pubspec.yaml` as usual.
 
 [1]: https://pub.dev/packages/url_launcher
 [2]: https://flutter.dev/docs/development/packages-and-plugins/developing-packages#endorsed-federated-plugin
+
+## Limitations on the Web platform
+
+### A launch needs to be triggered by a user action
+
+Some web browsers, such as Safari, may prevent URL launching if it is not triggered by a user action (e.g. a button click).
```

Comment:
Nit: We generally try to wrap our markdown files at about 80 characters; could you wrap all of these long lines?